mm2src/rpc/dispatcher/dispatcher_v2.rs

```diff
@@ -105,6 +105,7 @@ async fn dispatcher(request: MmRpcRequest, ctx: MmArc) -> DispatcherResult<Respo
         "enable_slp" => handle_mmrpc(ctx, request, enable_token::<SlpToken>).await,
         "get_public_key" => handle_mmrpc(ctx, request, get_public_key).await,
         "get_staking_infos" => handle_mmrpc(ctx, request, get_staking_infos).await,
+        "recover_swap" => handle_mmrpc(ctx, request, recover_swap).await,
```

Comment:
`recover_swap` looks too similar to `recover_funds_of_swap` - it may cause confusion and misunderstanding. I think that `recreate_swap_json` matches better in this case.

Reply:
How about `recreate_swap_data`?

Reply:
> recreate_swap_data

That also looks good to me :slightly_smiling_face:

Reply:
Done